superset/app.py

```diff
@@ -551,6 +553,7 @@ def init_app(self) -> None:
         self.pre_init()
         # Configuration of logging must be done first to apply the formatter properly
         self.configure_logging()
+        self.configure_db_enc()
```

Comment:
```suggestion
        self.configure_db_encrypt()
```

Because `enc` can often be confused with `encoding`.